src/tiledimage.js

```diff
@@ -1842,24 +1838,19 @@ $.extend($.TiledImage.prototype, $.EventSource.prototype, /** @lends OpenSeadrag
 
     /**
      * @private
-     * @inner
      * Obtains a tile at the given location.
      * @param {Number} x
      * @param {Number} y
      * @param {Number} level
      * @param {Number} time
      * @param {Number} numTiles
-     * @param {Number} worldWidth
```

Comment:
Good catch (that these aren't used). You should remove them from the place `_getTile` is called as well.

Reply:
Done